Voycar.Api.Web/Features/Users/Endpoints/Post/Login/Endpoint.cs

```diff
@@ -24,20 +24,18 @@ public override void Configure()
 
     public override async Task HandleAsync(Request req, CancellationToken ct)
     {
-        var user = await this._userRepository.RetrieveByEmail(req.Email!.ToLowerInvariant());
+        var user = await this._userRepository.RetrieveByEmail(req.Email.ToLowerInvariant());
 
 
         // Check if user entered valid credentials and verified his email
-        if (user is null || user!.VerifiedAt is null ||
-            !BCrypt.Net.BCrypt.EnhancedVerify(req.Password, user.PasswordHash))
+        if (user?.VerifiedAt is null || !BCrypt.Net.BCrypt.EnhancedVerify(req.Password, user.PasswordHash))
```

Comment:
Warum braucht man das "user is null" nicht mehr?

Reply:
Idk, Rider hat diese Changes vorgeschlagen, machen wohl angeblich dasselbe

Reply:
Rider hat recht. Mit dem null-conditional operator `?.` wird V`erifiedAt` nur ausgeführt, wenn user nicht `null` ist. Wenn user bereits `null` ist wird `null` zurückgegeben.

Reply:
Ah ok, ja dann passt das ja so